apps/api/src/app/notification-template/e2e/delete-notification-template.e2e.ts

```diff
@@ -139,6 +145,47 @@ describe('Delete notification template by id - /notification-templates/:template
 
     expect(response.body.message).to.contains('Could not find workflow with id');
   });
+
+  it('should delete the notification template along with the message templates', async function () {
```

Comment:
What do you think about also adding a test of deleting a notification template that has a feed assigned to it and then see that the feed can be deleted?

Reply:
Agree, that will be a good addition, will add it.